sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/uninstall/UninstallScheduler.java

```diff
@@ -182,20 +182,18 @@ protected void processOfferSet(List<Protos.Offer> offers) {
 
     @Override
     public void statusUpdate(SchedulerDriver driver, Protos.TaskStatus status) {
-        statusExecutor.execute(() -> {
-            LOGGER.info("Received status update for taskId={} state={} message='{}'",
-                    status.getTaskId().getValue(),
-                    status.getState().toString(),
-                    status.getMessage());
-
-            try {
-                stateStore.storeStatus(status);
-                reconciler.update(status);
-            } catch (Exception e) {
-                LOGGER.warn("Failed to update TaskStatus received from Mesos. "
-                        + "This may be expected if Mesos sent stale status information: " + status, e);
-            }
-        });
+        LOGGER.info("Received status update for taskId={} state={} message='{}'",
```

Comment:
This log message prints a different amount of information than that for the `DefaultScheduler`. Is that intentional?

If it is important to have this kind of logging uniform, one option would be to have `statusUpdate` implemented as a template method in the parent calling hooks (e.g. `logStatusUpdate`, `processStatusUpdate`) to do the real work and logging.